app/views/users/index.html.slim

```diff
@@ -0,0 +1,12 @@
+p#notice = notice
+h1
+  | Users
+
+.container
+  - @users.each do |user|
+    .card
```

Comment:
you can put code for each user card in file _user.html.slim and write at line 6

= render @users

magic)